src/common/localization/en-us.ts

```diff
@@ -80,6 +79,9 @@ export const english: IAppStrings = {
             frameExtractionRate: "Frame Extraction Rate (frames per a video second)",
         },
         addConnection: "Add Connection",
+        messages: {
+            saveSuccess: "Successfully saved ${project.name} project settings",
```

Comment:
Cool, I'm sure I'll see later on in this review, but I assume you're doing a similar thing with formatting the string as you did on the localization stuff previously? Clever